mesa/space.py

```diff
@@ -502,8 +503,8 @@ def position_agent(
         self, agent: Agent, x: int | str = "random", y: int | str = "random"
     ) -> None:
         """Position an agent on the grid.
-        This is used when first placing agents! Use 'move_to_empty()'
-        when you want agents to jump to an empty cell.
+        This is used when first placing agents! Setting either x or y to "random"
+        gives the same behavior as 'move_to_empty()'.
         Use 'swap_pos()' to swap agents positions.
         If x or y are positive, they are used, but if "random",
```

Comment:
This sentence seems to be slightly redundant because it explains the "random" x or y the second time. It should be merged with the "setting either x or y ..." and also that sentence should briefly say that `move_to_empty` puts you in a random position.

Reply:
[3c8c45f](https://github.com/projectmesa/mesa/pull/1457/commits/3c8c45f55bfbd1bbec4d690c91dd474842584429) - Update docstring to delete redundant expression